contracts/season3/TalentSponsorship.sol

```diff
@@ -50,14 +54,20 @@ contract TalentSponsorship {
         string symbol
     );
 
-    constructor() {
+    constructor(address contractOwner) {
         totalSponsorships = 0;
         totalRevokedSponsorships = 0;
         totalClaimedSponsorships = 0;
+        transferOwnership(contractOwner);
+    }
+
+    modifier notDisabled() {
+        require(!disabled, "The contract is currently disabled.");
```

Comment:
Let's change the message to: `The contract is disabled.` or a variation of this so it uses <= 32bits. This way we decrease a little bit the contract size

Reply:
Why is it relevant to decrease the contract size? Gas?

Reply:
Yes, that's the most important part

Reply:
Changed